calculator.rb

```diff
@@ -0,0 +1,67 @@
+print "Welcome to my calculator program! Please choose an operation 
+you would like to calculate\n"
+
+#Array with operations available
+#At some point figure out how to print directly from array so as to not repeat myself with puts
+# Change this into an array so a user could input other mathematical operations
+operations = ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"]
+
+puts "1. add(+) "
+puts "2. substract(-)"
+puts "3. mulitply(*)"
+puts "4. divide(/)"
+
+# Ask user to describe what operation performed. YOu will have to build in a feature that 
+#creates a way to check user input for correct format.
+
+print "Please choose an operator (name or symbol)"
+user_choice = gets.chomp
+choice = false 
+while choice == false
+    if operations.include?(user_choice)
+        choice = true
+        puts "success"
+    else
+       puts "Please enter a correct operation"
+       user_choice = gets.chomp
+    end
+end
```

Comment:
I like your strategy of using `choice` here in this while loop! It looks like `choice` will always be a boolean, and it will be `false` if the operator is invalid, and `true` if it is valid and successful. Here, you use a `while` loop AND an `if/else` statement. Could we use the `while` loops condition and the `if` condition, and consider combining them? Consider something like:

```ruby
while !operations.include?(user_choice)
  puts "Please enter a correct operation"
  user_choice = gets.chomp
end
```

This is saying "while the `operations` array does NOT include the value of `user_choice`, then that means that the `user_choice` is not valid, and so we ask for more user input. If the `operations` array DID include `user_choice`, then it was valid, and the computer will move past this code"